das/restful_server.go

```diff
@@ -55,26 +55,62 @@ func NewRestfulDasServerOnListener(listener net.Listener, storageService arbstat
 }
 
 type RestfulDasServerResponse struct {
-	Data string `json:"data"`
+	Data             string `json:"data"`
```

Comment:
I think we want to add `,omitempty` to these tags.

Reply:
Done.